README.md

```diff
@@ -17,18 +17,18 @@ Other terms used below:
 A minimalistic and stateless version of dusk-blockchain node where Consensus protocol in conjuction with Kadcast is able to join and participate `dusk-blockchain`/test-harness. Useful for testing and diagnostic incompatibility issues. This executable should be deprecated once dusk-blockchain is fully migrated.
 
 ## Example Testbed
-A multi-instance setup running 10 SBA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
+A multi-instance setup running 10 SA instances provisioned with 10 eligible participants. The setup is configured to run for up to 1000 rounds. Useful for any kind of testing (issues, stress and performance testing).
 
 ## Consensus crate
-A full implementation of SBA mechanism.
+A full implementation of SA mechanism.
 
  # Implementation details 
-SBA is driven basically by two tokio-rs tasks - Main Loop and Agreement Loop. Each of them has its own inbound and outbound message queues/channels to exchange messages with outsiders. SBA protocol parameters are defined in `/src/config.rs`.
+SA is driven basically by two tokio-rs tasks - Main Loop and Agreement Loop. Each of them has its own inbound and outbound message queues/channels to exchange messages with outsiders. SA protocol parameters are defined in `/src/config.rs`.
```

Comment:
I'd remove 'basically' and I'd say "two main tokio-rs tasks" (as there are more).
I'd change outsiders with external components or other components (or sth similar). Outsiders sounds weird to me

Reply:
Noted